bom/pom.xml

```diff
@@ -23,4 +23,62 @@
     <module>openhab-addons</module>
   </modules>
 
+  <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-antrun-plugin</artifactId>
+          <version>1.8</version>
+          <inherited>false</inherited>
+          <executions>
+            <execution>
+              <id>create-bom</id>
+              <goals>
+                <goal>run</goal>
+              </goals>
+              <configuration>
```

Comment:
Can we link some phase here to put in the right location of the build cycle or is this a deliberate choice.

Reply:
Well, we can't run that during our normal build (see above). If we have to execute it manually as seperate task anyway, we don't need a phase. I could add it to `generate-sources`, if you think that is useful.